discord/discord_video_source.cc

```diff
@@ -251,11 +251,13 @@ std::unordered_map<
 }  // namespace blink
 
 extern "C" {
-__declspec(dllexport) void DeliverDiscordFrame(
-    const char* streamId,
-    const blink::DiscordFrame& frame,
-    blink::DiscordFrameReleaseCB releaseCB,
-    void* userData) {
+#ifdef OS_WINDOWS
```

Comment:
figured we could wait until next time to put this in its own macro

Reply:
I was going to say there's probably a macro for this in chromium already, but it looks like they don't have a generic one. Just a bunch of specific ones for different modules that switch between dllexport and dllimport depending on whether a certain IMPLEMENTATION macro is defined